mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/property/ComposedOperationModelProperty.java

```diff
@@ -12,7 +12,9 @@
  * {@link ModelProperty} for indicating that the owning operation is a composed operation.
  *
  * @since 1.5
+ * @deprecated to be removed
  */
+@Deprecated
```

Comment:
why?

Reply:
this was only used in extensions-support, so i pushed this down there. Will be removed on follow-up pr.